kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewContent.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+
+  <div>
+    <div class="channel-header">
+      <div>
+        <h6>
```

Comment:
Should this be a heading tag? Specifically an `h6`? :sweat_smile: in the other subpages we are not treating it as a header, but we probably missed something :sweat_smile:.

We usually have this indicator for heading tags, but I dont see any in the figma

![image](https://github.com/user-attachments/assets/20b58929-0659-4ff8-8960-874f78392e20)

Reply:
It's something I saw here?

<img width="1197" alt="Screenshot 2025-02-05 at 16 19 05" src="https://github.com/user-attachments/assets/5d97ff5f-2141-4366-b0d4-75783ff3a678" />

Reply:
Oh, well it makes sense to me that you would make this an h6 then! But, I think that is actually an error in the Figma. I'll mention to Jessica and Tomiwa to be mindful of using header typography to get a certain font size in the spec. Thank you for raising this, @AllanOXDi - I hadn't noticed it. 

So, what should happen here: 
- the text doesn't actually need to be a header, it can just be a p (I'll confirm with @radinamatic)
- it should be _visually_ the same as in the figma (same font size and weight) which you can manage with css

Sorry for this confusion Allan! We can definitely improve on this going forward.

Reply:
Yes, confirmed with Radina, this doesn't need to be a heading. So for this page, "Manage lesson resources" would be the H2, and the title of the resource would be an H2. Everything else can be a p. 

<img width="696" alt="Screenshot 2025-02-05 at 9 56 44 AM" src="https://github.com/user-attachments/assets/9ad770b9-22c3-449d-8fa6-b93aa76de1a3" />